example/android/build.gradle

```diff
@@ -1 +1,10 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    repositories {
+        jcenter()
```

Comment:
We should probably start using `mavenCentral()`: https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

Reply:
I will resolve this for now, as switching to mavenCentral breaks ADP.